crypto/ledger_integration_test.go

```diff
@@ -0,0 +1,163 @@
+// +build cgo,ledger,real_ledger
```

Comment:
What is the difference between the `ledger` and `real_ledger` build targets? When use one over the other? Can we consolidate the two?

Reply:
For the first two, there are other files that depend on them:
`cgo`   to indicate that cgo is available
`ledger` it has been used historically in the SDK to indicate that support for ledger devices should be enabled and compiled.

`real_ledger` is to enable tests specific test that require a real device to run. 
This follows the same pattern as `cli_test`
https://github.com/cosmos/cosmos-sdk/pull/3431/files#diff-b67911656ef5d18c4ae36cb6741b7965R147

Maybe I can rename them to:
`cli_test` >> `test_cli_enabled`
`real_ledger` >> `test_ledger_enabled`

or something similar?